src/audio/Kconfig

```diff
@@ -58,7 +58,7 @@ config COMP_CHAIN_DMA
 
 config XRUN_NOTIFICATIONS_ENABLE
```

Comment:
the commit title is not really correct: "ace: kconfig: ..." this affects all IPC4 configs. please use "audio: ipc4: enable xrun notifications by default"